### ISSUE TITLE
Question : What does each if dimension of 4D array returned by net.forward()  in OpenCV dnn means? 

### ISSUE THREAD

**Issue by splash1123** [PyImageSearch/Deep-Learning-for-Computer-Vision-with-Python#5]
_Please use this template while reporting bugs inside DL4CV. Issues related to personal project questions or code that we haven't authored will automatically be closed._

## Are you using one of our development environments? 

We provide a preconfigured AWS AMI and a VM to help our customers skip the environment setup part and jump straight to development. We highly suggest using one of our development environments to run our code. 

If the issue is not related to code, then you can simply skip this question. 

## If you are using your own development environment then please provide the following details.

* GPU model and CUDA version: (skip if not application) 
* OS Platform: 
* Python version: 
* OpenCV version: 
* TensorFlow version:

## Describe the issue 
Hi, 
i recently tried COVID-19: Face Mask Detector with OpenCV, Keras/TensorFlow, and Deep Learning in pyiamgesearch's blog and was confused about What each if dimension of 4D array returned by net.forward()  in OpenCV dnn means. But i can not find the clear answer of the question in dl4cv books.

Can you please explain that for me?
Thank you very in advance!

Best, 
Drew Yang

**Comment by splash1123** [comment]
Here is the array returned by net.forward()
detections = net.forward()
detections.shape
#Out[~]: (1, 1, 200, 7)

**Comment by sayakpaul** [comment]
* First dimension denotes the batch size.
* You can ignore the second dimension. It's just a channel as the network is full-convolutional. 
* Third dimension denotes the number of detected objects. 
* For the last dimension:
    * Second index denotes the class index. 
    * Third index denotes the probability associated with the prediction.
    * Rest of the indices denote bounding box coordinates. 
    * I am not sure about the first index. So, apologies for that.
